data-processing/common/commands/expand_macros.py

```diff
@@ -0,0 +1,191 @@
+import logging
```

Comment:
consider putting a blurb at the top here, maybe using info from your pr description, about what macros are and why they need to be expanded

Reply:
Good thinking!